public/components/FeatureAnywhereContextMenu/CreateAnomalyDetector/helpers.tsx

```diff
@@ -43,7 +43,8 @@ const getFeatureNameFromVisParams = (id, seriesParams) => {
     }
   });
 
-  return name.data.label.replace(/[^a-zA-Z0-9-_]/g, '_');
+  const formaedFeatureName = name.data.label.replace(/[^a-zA-Z0-9-_]/g, '_');
```

Comment:
nit: formaedFeatureName -> formatedFeatureName.
Also surprised this didn't work before

Reply:
corrected the typo